changelog_unreleased/vue/pr-8747.md

```diff
@@ -0,0 +1,17 @@
+#### Fix parenthesized interpolation format ([#8747](https://github.com/prettier/prettier/pull/8747) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```vue
+<!-- Input -->
+<template>
+<span>{{(a||          b)}} {{z&&(a&&b)}}</span>
+</template>
+
+<!-- Prettier stable -->
+TypeError: Cannot set property 'undefined' of undefined
```

Comment:
It's a playground-only error.

Reply:
Updated https://github.com/prettier/prettier/pull/8747/commits/2e10a5b0e5f5d56ca66f18222c679c35b3b36160